src/main/java/pl/zaprogramujzycie/mma/config/TestSecurityConfig.java

```diff
@@ -0,0 +1,67 @@
+package pl.zaprogramujzycie.mma.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+// @EnableGlobalMethodSecurity(
+//         securedEnabled = true
+// )
+@EnableWebSecurity
+// @Profile("test")
+public class TestSecurityConfig {
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(final HttpSecurity http) throws Exception {
+        http.headers().frameOptions().disable();
+        return http.authorizeHttpRequests()
+                .antMatchers("/medicines/**")
+                .hasRole("MEDICINE-OWNER")
```

Comment:
authenticated czy cos takiego nie zalatwilo by sprawy?

Reply:
Pewnie by załatwiło, ale to jest tylko pod testy i musi mi przepuszczać niezalogowanych userów z konfiguracji poniżej.